cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/ACLineSegmentConversion.java

```diff
@@ -157,4 +141,21 @@ private static TieLine createTieLine(Context context, String boundaryNode, Bound
         }
         return tieLine;
     }
+
+    private static DanglingLineAdder getDanglingLineAdder(Context context, String boundaryNode, BoundaryLine boundaryLine) {
+        return context.network().getVoltageLevel(boundaryLine.getModelIidmVoltageLevelId())
+                .newDanglingLine()
+                .setId(boundaryLine.getId())
+                .setName(boundaryLine.getName())
+                // We consider p0 and q0 are equal to zero because there is no possible way to retrieve the corresponding
+                // equivalent injection. Note that the equivalent injections of both sides are compensating each other
+                // (the sum of their flows is supposed to be equal to zero).
```

Comment:
Beyond the scope of this PR but it makes me think that it could be of interest to log/report the difference of P0/Q0 in the merge function.